projects/uitdatabank/models/common-priceInfo.json

```diff
@@ -12,6 +12,43 @@
   },
   "minContains": 1,
   "maxContains": 1,
+  "examples": [
+    [
+      {
+        "category": "base",
+        "price": 10.5,
+        "priceCurrency": "EUR",
+        "name": {
+          "nl": "Basistarief",
+          "fr": "Tarif de base",
+          "en": "Base tariff",
+          "de": "Basisrate"
+        }
+      }
+    ],
+    [
+      {
+        "category": "base",
+        "price": 10.5,
+        "priceCurrency": "EUR",
+        "name": {
+          "nl": "Basistarief",
+          "fr": "Tarif de base",
+          "en": "Base tariff",
+          "de": "Basisrate"
+        }
+      },
+      {
+        "category": "tariff",
+        "price": 8,
+        "priceCurrency": "EUR",
+        "name": {
+          "nl": "Jongeren (<18j)",
+          "en": "Youth (<18yo)"
+        }
+      }
+    ]
+  ],
```

Comment:
Please discard changes like these in git whenever Stoplight re-shuffles properties, as it results in a confusing diff